benchmarking/platforms/host/host_platform.py

```diff
@@ -22,17 +23,23 @@
 
 class HostPlatform(PlatformBase):
     def __init__(self, tempdir):
-        self.setPlatformHash(str(socket.gethostname()))
+        platform_hash = str(socket.gethostname())
         if getArgs().platform_sig is not None:
-            self.setPlatform(str(getArgs().platform_sig))
+            platform = str(getArgs().platform_sig)
         else:
-            self.setPlatform(platform.platform() + "-" +
-                             self._getProcessorName())
-        self.tempdir = os.path.join(tempdir, self.platform + '_' +
-                                    str(self.platform_hash))
-        os.makedirs(self.tempdir, 0o777)
-        hdb = HDB(self.platform_hash)
+            platform = platform.platform() + "-" + \
+                               self._getProcessorName()
+        self.tempdir = os.path.join(tempdir, platform + '_' +
```

Comment:
One issue of using platform directly here is that it may contain white space or other letters not very good for filenames.